.gitignore

```diff
@@ -3,3 +3,6 @@
 /node_modules
 /vendor
 .php_cs.cache
+
+composer.lock
+package-lock.json
```

Comment:
This will keep diffs simple for this small site.

Reply:
These files make installation more reliable - it won't fail in the future even if I don't update the site. So I'd like to keep them I think.

Reply:
Sure.

Reply:
Is there a way to disable them from diffs in Github PRs?

It's 100 lines of changes and 3000 lines of locks.

Reply:
There isn't as far as I know. I actually did some research about it if it was possible to do using .gitattributes or some other meta files but I was only able to find people asking the same question but no solution.